apps/authentication/routes/__init__.py

```diff
@@ -1 +0,0 @@
-��
```

Comment:
Are these files having the right encodings?

Reply:
yes. Should be the action runner which was there